### PR TITLE
Fix merge hell

### PR DESCRIPTION
**Pull request by sargun**

### CODE REVIEW

**Comment by sargun** [comment]
@aeyakovenko

**Comment by sargun** [comment]
@aeyakovenko mind giving this a look see?

File: include/gen_netlink.hrl [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+-define(IP_RECVOPTS,			6).
+-define(IP_RETOPTS,			7).
+-define(IP_PKTINFO,			8).
+-define(IP_PKTOPTIONS,			9).
+-define(IP_PMTUDISC,			10).
+-define(IP_MTU_DISCOVER,		10).
+-define(IP_RECVERR,			11).
+-define(IP_RECVTTL,			12).
+-define(IP_RECVTOS,			13).
+-define(IP_MTU,				14).
```
Comment (aeyakovenko) [inline include/gen_netlink.hrl:182]: why is alignment screwed up?  tabs and spaces?

File: src/netlink_codec.erl [hunk 2]
```diff
@@ -0,0 +1,967 @@
...
+    Data.
+
+encode_genl_ctrl_attr_ops(Family, {Idx, Value}) ->
+    enc_nla(Idx, nl_enc_nla(Family, fun encode_genl_ctrl_attr_op/2, Value)).
+
+encode_genl_ctrl_attr_mcast_groups(Family, {Idx, Value}) ->
+    enc_nla(Idx, nl_enc_nla(Family, fun encode_genl_ctrl_attr_mcast_grp/2, Value)).
+
+encode_ipvs_service_attributes({flags, Flags, Mask}) ->
+    <<Flags:32/native-integer, Mask:32/native-integer>>.
```
Comment (aeyakovenko) [inline src/netlink_codec.erl:421]: do we need tests for this?

File: priv/decode_gen.escript [hunk 3]
```diff
@@ -5,860 +5,1034 @@
...
+            {supress_ifgroup, huint32},
+            {supress_prefixlen, huint32},
+            {table, huint32},
+            {fwmask, huint32},
+            {oifname, string},
+            {pad, none},
+            {l3mdev, uint8}
+        ]},
+
+        {{nl_msgtype, queue}, [
```
Comment (aeyakovenko) [inline priv/decode_gen.escript:571]: should we boot the nfq codec?

File: include/gen_netlink.hrl (hunk 1 above)
Comment (sargun) [reply include/gen_netlink.hrl:182]: It looks that way, this is code from upstream that's been reformatted / moved around.

File: src/netlink_codec.erl (hunk 2 above)
Comment (sargun) [reply src/netlink_codec.erl:421]: There are tests, see test_ipvs in ct.

File: priv/decode_gen.escript (hunk 3 above)
Comment (sargun) [reply priv/decode_gen.escript:571]: Nah, no reason to ever remove these, IMHO.

**Pull request merged**